test/files/run/caseClassEquality.scala

```diff
@@ -2,7 +2,9 @@ object Test {
   abstract class A1
   case class C1(x: Int) extends A1
   class C2(x: Int) extends C1(x) {
-    override def productPrefix = "Shazbot!"
+    // this used to be a different prefix, not sure what it was testing
+    // (now, hashCode takes the prefix into account, so using the same as the super case class is the only way to maintain the assert on hashCode below)
+    override def productPrefix = "C1"
```

Comment:
It looks like the `productPrefix` is `C1` without overriding it :thinking:

Reply:
ah yeah, because C2 is not a case class -- I was pretty confused by this test and what it was really about